bitrise.yml

```diff
@@ -160,4 +170,4 @@ workflows:
                 Workflow|${BITRISE_TRIGGERED_WORKFLOW_ID}
             - buttons: |-
                 View on Bitrise|${BITRISE_BUILD_URL}
-                View on HockeyApp|https://rink.hockeyapp.net/apps/${HOCKEYAPP_APP_ID}
+                View on App Center|https://appcenter.ms/orgs/${APP_CENTER_ORG_NAME_ID}/apps/${APP_CENTER_APP_NAME_ID}
```

Comment:
You can also update link in ReadMe.
Currently there is a link to HockeyApp - staging

Reply:
Good catch!